README.md

```diff
@@ -57,10 +57,13 @@ const cache = new InvalidationPolicyCache({
 | `modify`                      | `modify` API from Apollo cache     |
 | `readField`                   | `readField` API from Apollo cache  |
 
-| Extended cache API | Description                                                                                | Return Type                                                   |
-| -------------------| -------------------------------------------------------------------------------------------|---------------------------------------------------------------|
-| `expire`           | Evicts all expired entities from the cache based on their type's or the global timeToLive. | String[] - List of expired entity IDs evicted from the cache. |
-| `expiredEntities`  | Returns all expired entities still present in the cache                                    | String[] - List of expired entities in the cache.             |
+| Extended cache API       | Description                                                                               | Return Type                                                  | Arguments                 |
+| -------------------------| ------------------------------------------------------------------------------------------|--------------------------------------------------------------|---------------------------|
+| `expire`                 | Evicts all expired entities from the cache based on their type's or the global timeToLive | String[] - List of expired entity IDs evicted from the cache | N/A                       |
+| `expiredEntities`        | Returns all expired entities still present in the cache                                   | String[] - List of expired entities in the cache             | N/A                       |
+| `activePolicyEvents`     | Returns all active policy events (Read, Write, Evict)                                     | InvalidationPolicyEvent[] - List of active policy events     | N/A                       |
+| `activatePolicyEvents`   | Activates the provided policy events, defaults to all                                     | void                                                         | InvalidationPolicyEvent[] |
```

Comment:
The docs suggest that the method takes one argument that's an array, but the code and tests show that the method takes any number of arguments of type `InvalidationPolicyEvent`. What's the intended API?

```js
const policyEvents = [
  InvalidationPolicyEvent.Read,
  InvalidationPolicyEvent.Write
];

cache.activatePolicyEvents(policyEvents);

// or

cache.activatePolicyEvents(...policyEvents);
```

If the latter I suggest maybe defining the argument list as `...InvalidationPolicyEvent[]` instead of `InvalidationPolicyEvent[]`. I can't remember seeing another example of arbitrary-length arguments, so there might be an established way of communicating that it's not a single argument.

Reply:
Ah nice catch! Ya I should have put `...InvalidationPolicyEvent[]` in the docs!